languages/python/oso/polar/polar.py

```diff
@@ -86,6 +87,16 @@ def __del__(self):
         del self.host
         del self.ffi_polar
 
+    def enable_roles(self):
+        class InternalRolesHelpers:
+            @staticmethod
+            def join(separator, left, right):
+                return separator.join([left, right])
+
+        self.register_constant(InternalRolesHelpers, "__oso_internal_roles_helpers__")
+        self.ffi_polar.enable_roles()
+        self._polar_roles_enabled = True
```

Comment:
Should we check if this is already `True` to prevent them calling the method twice by accident?

Reply:
Sure, everything's already idempotent, but doing the explicit check will make that more obvious

Reply:
5ba7d14df62b091354fdf8b7f30e02c8bd17595a